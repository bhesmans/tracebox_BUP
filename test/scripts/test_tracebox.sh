#!/bin/bash

/usr/local/bin/tracebox -y -n 123.123.123.2

