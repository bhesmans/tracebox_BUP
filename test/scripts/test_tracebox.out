tracebox to 123.123.123.2 (123.123.123.2): 30 hops max
 1: 42.42.42.1 
 2: 123.123.123.2 
